tests/api/benchmark/test_checkout_mutations.py

```diff
@@ -0,0 +1,991 @@
+import pytest
+from graphene import Node
+
+from saleor.checkout import calculations
+from saleor.checkout.models import Checkout
+from tests.api.utils import get_graphql_content
+
+
+@pytest.mark.django_db
+@pytest.mark.count_queries(autouse=False)
+def test_create_checkout_as_anonymous(
+    api_client,
+    graphql_address_data,
+    stock,
+    product_with_default_variant,
+    product_with_single_variant,
+    product_with_two_variants,
+    count_queries,
+):
+    query = """
+      fragment Price on TaxedMoney {
```

Comment:
Don't you want to extract these fragments to some constants and reuse them? For example, this one is defined six times in this module. I would try to extract it and reuse using Python's template strings to construct queries.

Reply:
It's a copy of a query from the storefront. 
When some part of storefront query will change it's easier to replace the whole query instead od looking for diffs. 
I can split it to parts if you want?

Reply:
After a small discussion, we decided, to move fragment to const.
In the feature, we should create fragments with all field of type and reuse it as many time as possible.

Reply:
Some fragments like addresses or prices could be extracted anyway. I would do that now in this file and later we'll see if we can reuse that in other files as well (but that's a separate PR).